quickstart.rst

```diff
@@ -120,36 +120,36 @@ In order to use Flux, you first must initiate a Flux *instance* or *session*.
 
 A Flux session is composed of a hierarchy of ``flux-broker`` processes which are launched via any parallel launch utility that supports PMI. For example, ``srun``, ``mpiexec.hydra``, etc., or locally for testing via the ``flux start`` command.
 
+Before a Flux instance can be started, keys must be generated to encrypt and authenticate Flux messages.  This step is only required for first-time users of flux.
+
+.. code-block:: console
+
+  $ flux keygen
+  Saving /g/g0/grondo/.flux/curve/client
+  Saving /g/g0/grondo/.flux/curve/client_private
+  Saving /g/g0/grondo/.flux/curve/server
+  Saving /g/g0/grondo/.flux/curve/server_private
```

Comment:
Not that I mind the notoriety, but maybe we should generalize  the username used throughout the quickstart guide. (below a different username is used for the jobs list).